src/models/tests/m_rsc_db_tests.erl

```diff
@@ -11,8 +11,8 @@ modify_rsc_test() ->
     AdminC = z_acl:logon(?ACL_ADMIN_USER_ID, C),
     CatId = m_rsc:rid(text, C),
 
-    ?assertThrow({error, eacces}, m_rsc:insert([{title, "Hello."}], C)),
-    ?assertThrow({error, eacces}, m_rsc:insert([{title, "Hello."}, {category_id, CatId}], C)),
+    ?assertThrow({{error, eacces}, _Trace}, m_rsc:insert([{title, "Hello."}], C)),
```

Comment:
@mworrell I’m not sure this is what we want to return; wouldn’t just `{error, eacces}` be better?